wpilibj/src/main/java/edu/wpi/first/wpilibj/kinematics/SwerveModuleState.java

```diff
@@ -42,6 +42,21 @@ public SwerveModuleState(double speedMetersPerSecond, Rotation2d angle) {
     this.angle = angle;
   }
 
+  /**
+   * Optimizes the angle of the module to make sure it doesn't 
+   * rotate more than 90 degrees from the current state
+   */
+  public SwerveModuleState OptimizeModuleAngle(SwerveModuleState state) {
+    SwerveModuleState finalStateAfterOptimization = state;
```

Comment:
This does not copy the actual value, it simply copies the reference, meaning that any change to `finalStateAfterOptimization` will also result in changes to `state`.